src/DropDownMenu/DropDownMenu.js

```diff
@@ -274,15 +294,28 @@ class DropDownMenu extends Component {
 
   handleItemTouchTap = (event, child, index) => {
```

Comment:
I haven't realized that the `Menu` component already handles a `multiple` property.
With that in mind, why is `onItemTouchTap` triggering the `onChange` event. Should it be decoupled and handled aside? I mean in the `handleOnChange` function. I feel like we have quite some logic duplication.